ci/linux-packages/Rakefile

```diff
@@ -130,7 +130,7 @@ class ADBCPackageTask < PackageTask
   def apt_targets_default
     [
       "debian-bookworm",
-      # "debian-bookworm-arm64",
+      "debian-bookworm-arm64",
```

Comment:
We don't need this. Because the CI job specify the APT target explicitly instead of using the default targets.